01-node-tutorial/answers/content/first.txt

```diff
@@ -0,0 +1 @@
+Hello, this is my first file
```

Comment:
The temporary files should not be committed to github.  Make sure to put them in the temporary folder and add that folder to .gitignore